ssi-crypto/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "ssi-crypto"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = ["ring"]
```

Comment:
Should ring be default? It doesn't appear to be default in the other crates.

Reply:
changed defaults to `secp256k1` and `ripemd-160`